opendevin/core/config.py

```diff
@@ -195,6 +195,7 @@ class AppConfig(metaclass=Singleton):
     enable_auto_lint: bool = (
         False  # once enabled, OpenDevin would lint files after editing
     )
+    enable_main_session: bool = False
```

Comment:
My understanding is you name it "main" because it's only used by `main` function (not web GUI). If that's the case I propose we rename it to something like "enable_cli_session".